piv/src/main/java/com/yubico/yubikit/piv/PivSession.java

```diff
@@ -442,6 +455,7 @@ private byte[] usePrivateKey(Slot slot, KeyType keyType, byte[] message, boolean
      * @throws ApduException in case of an error response from the YubiKey
      */
     public void setManagementKey(ManagementKeyType keyType, byte[] managementKey, boolean requireTouch) throws IOException, ApduException {
+        logger.debug("Setting management key of type: {}", keyType);
```

Comment:
This class seems to mix `org.slf4j.Logger` and `com.yubico.yubikit.core.Logger`, is that intentional?